### PR TITLE
Solve issue #1 

### PR DESCRIPTION
**Pull request by kydos**

### CODE REVIEW

File: zenoh/src/main/java/org/eclipse/zenoh/Workspace.java [hunk 1]
```diff
@@ -72,6 +73,40 @@ private Selector toAsbsoluteSelector(Selector s) {
...
 
+    /**
+     * Put a path/value into Zenoh.
+     *
+     * @param path  the string representing the path. Can be absolute or relative to the workspace.
+     * @param value the value {@link java.lang.String}.
+     * @throws ZException if put failed.
+     */
+    public void put(String path, String value) throws ZException {
+        this.put(new Path(path), new StringValue(value));
```
Comment (JEnoch) [inline zenoh/src/main/java/org/eclipse/zenoh/Workspace.java:84]: We could avoid boxing of value within a StringValue replacing this line with a code similar to put(String, Value) implementation (i.e. directly calling session.write() with STRING encoding).
For this we would need to add a private static StringValue.encode(String) method.

File: zenoh/src/main/java/org/eclipse/zenoh/Workspace.java [hunk 2]
```diff
@@ -72,6 +73,40 @@ private Selector toAsbsoluteSelector(Selector s) {
...
+
+    /**
+     * Put a path/value into Zenoh.
+     *
+     * @param path  the string representing the path. Can be absolute or relative to the workspace.
+     * @param value the int value
+     * @throws ZException if put failed.
+     */
+    public void put(String path, int value) throws ZException {
+        this.put(new Path(path), new IntValue(value));
```
Comment (JEnoch) [inline zenoh/src/main/java/org/eclipse/zenoh/Workspace.java:95]: Same comment wrt. avoidable boxing of int within IntValue.

File: zenoh/src/main/java/org/eclipse/zenoh/Workspace.java [hunk 3]
```diff
@@ -72,6 +73,40 @@ private Selector toAsbsoluteSelector(Selector s) {
...
+    /**
+     * Put a path/value into Zenoh.
+     *
+     * @param path  the string representing the path. Can be absolute or relative to the workspace.
+     * @param value the float value.
+     * @throws ZException if put failed.
+     */
+
+    public void put(String path, float value) throws ZException {
+        this.put(new Path(path), new FloatValue(value));
```
Comment (JEnoch) [inline zenoh/src/main/java/org/eclipse/zenoh/Workspace.java:107]: Same comment wrt. avoidable boxing of int within FloatValue.

File: examples/zenoh-net/src/main/java/ZNStream.java [hunk 4]
```diff
@@ -16,7 +16,7 @@
 class ZNStream {
 
     public static void main(String[] args) {
-        String uri = "/demo/example/zenoh-java-stream";
+        String uri = "/zenoh/examples/java/stream";
```
[the remark this answers is not in the document]
Comment (kydos) [reply examples/zenoh-net/src/main/java/ZNStream.java:19]: That's a miss. Indeed, will fix it.

File: examples/zenoh-net/src/main/java/ZNWrite.java [hunk 5]
```diff
@@ -16,7 +16,7 @@
 class ZNWrite {
 
     public static void main(String[] args) {
-        String uri = "/demo/example/zenoh-java-write";
+        String uri = "/zenoh/examples/java/write";
```
[the remark this answers is not in the document]
Comment (kydos) [reply examples/zenoh-net/src/main/java/ZNWrite.java:19]: Needs the hello.

File: zenoh/src/main/java/org/eclipse/zenoh/Workspace.java (hunk 1 above)
Comment (kydos) [reply zenoh/src/main/java/org/eclipse/zenoh/Workspace.java:84]: Indeed, that would avoid creating the XYZValue.

**Pull request merged**